src/Symfony/Component/Notifier/Bridge/LineBot/Tests/LineBotTransportFactoryTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LineBot\Tests;
+
+use Symfony\Component\Notifier\Bridge\LineBot\LineBotTransportFactory;
+use Symfony\Component\Notifier\Test\AbstractTransportFactoryTestCase;
+use Symfony\Component\Notifier\Test\IncompleteDsnTestTrait;
+use Symfony\Component\Notifier\Test\MissingRequiredOptionTestTrait;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Yi-Jyun Pan <me@pan93.com>
+ */
+final class LineBotTransportFactoryTest extends AbstractTransportFactoryTestCase
+{
+    use IncompleteDsnTestTrait;
+    use MissingRequiredOptionTestTrait;
+
+    private const MOCK_TOKEN = 'eyJhbGciOiJIUzI1NiJ9.eyJSb2xlIjoiQWRtaW4iL+CJJc3N1ZXIiOiJJc3N1ZXIiLCJVc2VybmFtZSI6IkphdmFJblVzZSIsImV4cCI6MTcyODU1MjA3OSwiaW+F0IjoxNzI4NTUyMDc5fQ.SPKpGKwsXBay2uXDh7tATW20S2vZpw9qcmYjNp46Ir/AB/12345677=';
```

Comment:
Do we need that long token? Is it validated somewhere?

Reply:
Well, no.

The `MOCK_TOKEN` contains characters like `+` and `/`, which are unsafe in a URI, but are typically found in a LINE access token. I did not validate the token's validity at runtime because:

* LINE has not specified the format of an access token, making it impossible to create a regex for validation.
* LINE's access tokens include URL-unsafe strings, so I wrote some tests (not directly related to LineBot) to ensure the token can be encoded and decoded properly as expected.

Reply:
Lets keep it then, thanks for the explanation